.github/workflows/aws_deploy.yml

```diff
@@ -167,6 +167,7 @@ jobs:
             NEXT_PUBLIC_S3_ASSETS_DOMAIN_NAME=${{ inputs.app-s3-assets-domain-name }}
             NEXT_PUBLIC_S3_ASSETS_BUCKET_NAME=${{ inputs.app-s3-assets-bucket-name }}
             NEXT_PUBLIC_GROWTHBOOK_CLIENT_KEY:${{ secrets.GROWTHBOOK_CLIENT_KEY }}
+            NEXT_PUBLIC_INTERCOM_APP_ID:${{ secrets.INTERCOM_APP_ID }}
```

Comment:
just fyi, in order for this to work, we need to tell Docker that this is a build arg. in order to do that, we need to pass this as a build arg + env in docker!

just passing to docker isn't quite enough because docker doesn't know that this is supposed to be a build arg. hence, when we deploy this to envs with this secret, it won't quite work (but won't crash the app)

in order to get this picked up by docker as a build arg, we need `ARG NEXT_PUBLIC_INTERCOM_APP_ID` and for it to be available to the app as an env we also need to pass `ENV NEXT_PUBLIC_INTERCOM_APP_ID = $NEXT_PUBLIC_INTERCOM_APP_ID`

Reply:
@seaerchin okay done, should i do the same for `NEXT_PUBLIC_GROWTHBOOK_CLIENT_KEY` in another separate PR?